dist/index.js

```diff
@@ -305,6 +320,72 @@ class NixInstallerAction {
             return false;
         }
     }
+    async setup_kvm() {
+        const kvm_rules = "/etc/udev/rules.d/99-determinate-nix-installer-kvm.rules";
+        try {
+            const write_file_exit_code = await _actions_exec__WEBPACK_IMPORTED_MODULE_3__.exec("sh", [
+                "-c",
+                `echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee ${kvm_rules} > /dev/null`,
```

Comment:
Graham "Don't write bash" Christensen. 😏

Reply:
🤡 I didn't want to 😭